src/Uno.WinUIRevert/Program.cs

```diff
@@ -216,5 +218,13 @@ private static void DeleteFiles(string[] filePaths)
 				}
 			}
 		}
+
+		private static void AddWindowsDependencyGroup(string nuspecPath)
+		{
+			var contents = File.ReadAllText(nuspecPath);
+			contents = Regex.Replace(contents, @"<!-- <group targetFramework=""net5.0-windows10.0.18362.0"">", @"<group targetFramework=""net5.0-windows10.0.18362.0"">");
+			contents = Regex.Replace(contents, @"</group> -->", @"</group>");
```

Comment:
This could be a little bit surprising. One could comment something and endup with an invalid nuspec file.

I would do it with special marker, for example, in nuspec we would have the following:

<!-- BEGIN WinUI-specific
Here the winui-specific parts
END WinUI-specific -->

and here in WinUIRevert, we would be replacing `"<!-- BEGIN WinUI-specific"` with empty string and `"END WinUI-specific -->"` with empty string too.

Reply:
That way it is also a more general approach for more WinUI-specific stuff.